File: src/DataStructure/Array/1D/FizzBuzz/python/FizzBuzz_test.py
import pytest
from FizzBuzz.python.fizzBuzz import Solution
from FizzBuzz.python.testCases import runTestCase, cases


@pytest.mark.parametrize('case', cases)
def test(case):
    solution = Solution()
    assert runTestCase(solution.fizzBuzz, case) == case['expected']

File: src/DataStructure/Array/1D/Easy/FizzBuzz/javascript/fizzBuzz.test.js
import solution from './fizzBuzz.es3.cjs';
import runTestCase, { cases } from './testCases.js';

test.each(cases.map(c => [c.name, c]))(
  '[Data Structure/Array/1D/Easy/[412] Fizz Buzz] - Case %#',
  (name, testCase) => expect(runTestCase(solution, testCase)).toEqual(testCase.expected),
);
